Makefile

```diff
@@ -36,6 +38,14 @@ build: layout
 build-binary:
 	go build -o PolicyGenerator cmd/main.go
 
+build-so:
+	@mkdir -p build_output
+		go build -buildmode=c-shared -o build_output/_processGeneratorConfigC_$(OS)_$(ARCH).so cmd/main.go
+
+build-so-multiarch:
```

Comment:
Could you explain why this is `multiarch`? It seems that it's just producing an `amd64` SO.